packages/web/tests/helpers/event.ts

```diff
@@ -0,0 +1,6 @@
+const triggerEvent = (element: HTMLElement, eventName: string, options = { bubbles: true, cancelable: false }) => {
```

Comment:
Here is no need for this function. Please use `EventHandler.trigger(...)`

Reply:
fixed